src/styles/variables.less

```diff
@@ -44,6 +44,10 @@
 @color-gray-25: fade(#000, 25%);
 @color-gray-30: fade(#000, 30%);
 
+// Shadows
+
+@shadow-inset: 0 10px 10px -10px @color-gray-30 inset;
```

Comment:
question: does it make sense to name the variable `@box-shadow-inset` to be consistent with the css property?